### PR TITLE
Add ability to add blog post author

### PR DESCRIPTION
**Pull request by palmfjord**

### CODE REVIEW

**Comment by palmfjord** [comment]
`heroku addons:create heroku-redis` imo at least :) pulled it in because I want to utilize it for caching later

**Comment by palmfjord** [comment]
@hedvigoscar heroku redis is fairly cheap considering we store so little data in there. If not, heroku redis is just a wrapper around elasticache if i remember it correctly (but i may be wrong)

File: src/server/utils/redis.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as redis from 'redis'
```
Comment (hedvigoscar) [inline src/server/utils/redis.ts]: I've been told somewhere that `ioredis` is a good package. We 'implicitly' use it in giraffe, could be worth taking a look at?

Comment (palmfjord) [reply src/server/utils/redis.ts]: looks very neat! nice catch

Comment (palmfjord) [reply src/server/utils/redis.ts]: or "catch".. idk. input?

**Pull request merged**